packages/expo-av/android/CMakeLists.txt

```diff
@@ -16,65 +19,31 @@ add_library(
         src/main/cpp/JAVManager.cpp
 )
 
-# Extracted AAR: ${BUILD_DIR}/react-native-0*/jni/${ANDROID_ABI}
-file(GLOB LIBRN_DIR "${RN_SO_DIR}/${ANDROID_ABI}")
-if (NOT LIBRN_DIR)
-    # If /${ANDROID_ABI} dir not found, then ${RN_SO_DIR} is probably:
-    # ReactAndroid/build/react-ndk/exported
-    file(GLOB LIBRN_DIR "${RN_SO_DIR}")
-endif()
-
-file(GLOB libfbjni_include_DIRS "${BUILD_DIR}/fbjni-*-headers.jar/")
-
 # includes
 
 target_include_directories(
         ${PACKAGE_NAME}
         PRIVATE
-        "${REACT_NATIVE_DIR}/React"
-        "${REACT_NATIVE_DIR}/React/Base"
-        "${REACT_NATIVE_DIR}/ReactAndroid/src/main/jni"
+        # header only imports from turbomodule, e.g. CallInvokerHolder.h
         "${REACT_NATIVE_DIR}/ReactAndroid/src/main/jni/react/turbomodule"
-        "${REACT_NATIVE_DIR}/ReactAndroid/src/main/java/com/facebook/react/turbomodule/core/jni"
-        "${REACT_NATIVE_DIR}/ReactCommon"
         "${REACT_NATIVE_DIR}/ReactCommon/callinvoker"
```

Comment:
thanks @tomekzaw for the kind comment. i just don't want to introduce new linked libs to libexpo-av.so, as expo-av currently doesn't rely on runtimexecutor. although we can get the prefab include path through get_target_property, e.g.
```cmake
get_target_property(INCLUDE_runtimexecutor
        ReactAndroid::runtimeexecutor
        INTERFACE_INCLUDE_DIRECTORIES)
target_include_directories(
        ${PACKAGE_NAME}
        PRIVATE
        ${INCLUDE_runtimexecutor}
        # ...
)
```

since we only use the single header, it doesn't hurt to reference the header directly from react-native folder.

Reply:
Thanks @Kudo for the explanation! I really like the idea of `get_target_property`, gonna try it in Reanimated! 😄